proto/regen/ecocredit/v1/tx.proto

```diff
@@ -215,19 +224,111 @@ message MsgMintBatchCredits {
   string note = 5;
 }
 
-// MsgSealBatch request type for MsgSealBatch RPC.
+// MsgMintBatchCreditsResponse is the Msg/MintBatchCredits response type.
+message MsgMintBatchCreditsResponse {}
+
+// MsgSealBatch request type for SealBatch RPC.
 message MsgSealBatch {
   // Issuer must equal to the batch.issuer address.
   // Signer of the msg.
   string issuer = 1;
 }
 
-// MsgMintBatchCreditsResponse is the Msg/MintBatchCredits response type.
-message MsgMintBatchCreditsResponse {}
-
 // MsgSealBatchResponse is the Msg/SealBatch response type.
 message MsgSealBatchResponse {}
 
+// MsgMintPendingBatchCredits is the Msg/MintPendingBatchCredits request type.
+message MsgMintPendingBatchCredits {
+
+  // issuer is the address of the credit class issuer who is being requested
+  // to approve the batch issuance.
+  string issuer = 1;
```

Comment:
shouldn't this be Alice?

Reply:
No this would be Bob - Alice is the Verra credit holder requesting for a pending batch to be created. Bob is the issuer who will review and approve the pending batch